packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -46,4 +46,179 @@ export const components: ThemeOptions["components"] = {
+          },
+        },
+      },
+      {
+        props: { variant: "secondary" },
+        style: {
+          backgroundColor: "#f5f5f6",
+          borderColor: "#d7d7dc",
+          color: "#1d1d21",
+          "&:hover": {
+            background: "#f2f5ff",
+            borderColor: "#a7b5ec",
+            color: "#1662dd",
+          },
+
+          "&:focus-visible": {
+            backgroundColor: "#1662dd",
+            color: "#1662dd",
+          },
+
+          "&:active": {
+            borderColor: "#1662dd",
+          },
+
+          "&:disabled": {
+            borderColor: "#ebebed",
+            backgroundColor: "#ebebed",
+            color: "#8c8c96",
+          },
+        },
+      },
+      {
+        props: { variant: "danger" },
+        style: {
+          backgroundColor: "#da372c",
+          color: "#fff",
+          "&:hover": {
+            borderColor: "transparent",
+            backgroundColor: "#640019",
+          },
+
+          "&:focus-visible": {
+            outlineColor: "#f88c90",
+            backgroundColor: "#640019",
+          },
+
+          "&:active": {
+            borderColor: "transparent",
+            backgroundColor: "#da372c",
+          },
+
+          "&:disabled": {
+            color: "#fff",
+            borderColor: "#f88c90",
+            backgroundColor: "#f88c90",
+          },
+        },
+      },
+      {
+        props: { variant: "floating" },
+        style: {
+          backgroundColor: "#fff",
+          color: "#1d1d21",
+          borderColor: "transparent",
+
+          "&:hover, &:focus-visible": {
+            backgroundColor: "rgba(29, 29, 33, 0.1)",
+            borderColor: "transparent",
+          },
+          "&:active": {
+            backgroundColor: "rgba(29, 29, 33, 0.2)",
+            borderColor: "transparent",
+          },
+          "&:disabled": {
+            backgroundColor: "rgba(235, 235, 237, 0.6)",
+            color: "#6e6e78",
+            borderColor: "transparent",
+          },
+        },
+      },
+      {
+        props: { size: "s" },
+        style: {
+          paddingBlock: "calc(0.57142857rem - 1px)",
+          paddingInline: "calc(0.85714286rem - - 1px)",
+          fontSize: "1rem",
+        },
+      },
+      {
+        props: { size: "l" },
+        style: {
+          paddingBlock: "calc(1.14285714rem - 1px)",
+          paddingInline: "calc(1.14285714rem - 1px)",
+        },
+      },
+      {
+        props: { fullWidth: true },
+        style: {
+          display: "block",
+          width: "100%",
+          marginBlock: "0",
+          marginInline: "0",
+
+          "&:not(:last-child)": {
+            marginBlockEnd: "1.14285714rem",
+          },
+        },
+      },
+    ],
+    styleOverrides: {
+      root: ({ ownerState }) => ({
+        ...ownerState,
+        fontWeight: 600,
+        paddingBlock: "0.85714286rem",
+        paddingInline: "0.85714286rem",
+        fontFamily:
```

Comment:
`fontFamily` should have picked up from `typography` but it didn't

Reply:
I ran into this issue too. Sadly, even forcing a style override on `MuiTypography` doesn't cover all the bases, since some components don't rely on `TypographyRoot` 😞.  Since this seems to be a Storybook-centric issue, I [added these lines](https://github.com/okta/odyssey/pull/1499/files#diff-dbaca548056fac468c040d8e2ec8eff229150099e56879178b3f1ea2ff1dfa80R12) to the decorator until we can update SB.

Reply:
This also works:
```
  MuiTypography: {
    defaultProps: {
      fontFamily:
        "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
    },
```
looks like [others](https://github.com/mui/material-ui/issues/28633) have this issue too

Reply:
^ that last comment was added before I saw your comment @edburyenegren-okta

Reply:
In non-SB cases, this should cascade down from `typography` normally, right? If so, my vote is to pull this out of the Button theme and rely on our SB hack for now.